src/web/mjs/connectors/GuyaMoe.mjs

```diff
@@ -0,0 +1,134 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class GuyaMoe extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'guya-moe';
+        super.label = 'Guya.moe';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://manga.guya.moe';
+    }
+
+    async _getMangas() {
+        const request = new Request(this.url, this.requestOptions);
+        /** @type {HTMLDivElement[]} */
+        const [data] = await this.fetchDOM(request, '.dropdown-menu');
+        const anchors = data.querySelectorAll('a');
+        const result = [];
+        anchors.forEach(a => {
```

Comment:
May use `map` instead of pushing into result, e.g.
`const results = anchors.map(a => { /* return manga result instead of push */ })`